masterlist.yaml

```
@@ -3544,6 +3544,22 @@ plugins:
         condition: 'not (file("Particle Patch for ENB.bsa") and active("Particle Patch for ENB.esp")) and not file("textures/effects/gradients/gradillusiondark01m.dds") and not file("textures/WiZkiD/iron_e.dds")'
     tag: [ Graphics ]
 
+  - name: 'ImprovedFog&Lighting.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/46289/'
+        name: 'Improved Fog and Lighting'
+    group: *lightingEnhancerGroup
+    inc:
+      - 'ELE_SSE.esp'    
+      - 'ELFXEnhancer.esp'
+      - 'ELFX - Hardcore.esp'
+      - 'EnhancedLightsandFX.esp'
```

Both plugins have `EnhancedLightsandFX.esp` as a master, so listing it as incompatible is sufficient:

```suggestion
      - 'EnhancedLightsandFX.esp'
```